Sources/Danger/Plugins/SwiftLint/SwiftLint.swift

```diff
@@ -59,19 +59,24 @@ extension SwiftLint {
             violations = makeViolations(from: outputJSON, failAction: failAction)
         } else {
             // Gathers modified+created files, invokes SwiftLint on each, and posts collected errors+warnings to Danger.
-            var files = danger.git.createdFiles + danger.git.modifiedFiles
+            var files = (danger.git.createdFiles + danger.git.modifiedFiles).filter { $0.hasSuffix(".swift") }
```

Comment:
I think you can use $0.fileType, you can use https://github.com/artsy/eigen/blob/master/Dangerfile.swift#L16 as example